config.toml

```diff
@@ -25,6 +25,8 @@ recaptchaKey = "" # Google reCAPTCHA key (set $HUGO_PARAMS_RECAPTCHAKEY)
 intro = "WELCOME TO ROTATIONAL"
 
 # google fonts
+googleFontsApi = "https://fonts.googleapis.com"
```

Comment:
Google fonts now includes these links for preconnecting to improve loading so this will now be added to the `head` element.